app/src/main/java/sample/okta/android/SampleApplication.kt

```diff
@@ -38,12 +35,6 @@ class SampleApplication : Application() {
 
         Timber.plant(Timber.DebugTree())
 
-        Credential.Security.standard = Credential.Security.BiometricStrongOrDeviceCredential(userAuthenticationTimeout = 5)
-        Credential.Security.promptInfo = BiometricPrompt.PromptInfo.Builder()
-            .setTitle("Title")
-            .setNegativeButtonText("Cancel Button")
-            .setAllowedAuthenticators(BiometricManager.Authenticators.BIOMETRIC_STRONG) // Verify the authenticator is supported by device using BiometricManager.canAuthenticate
-            .build()
```

Comment:
This isn't a part of the gradle changes, but it was a testing change that was accidentally left in sample code.